submissions/egwoloOkorodudu/programmingLanguages1/programmingLanguage1.txt programmingLanguage1

```diff
@@ -0,0 +1,2 @@
+Research Programming languages 
+https://docs.google.com/spreadsheets/d/1RcLNZOFzpNjfKjZPhGgWdyDQI8Q7iU4Dej7yPZhCvtw/edit?usp=sharing
```

Comment:
Did you redo this task? If yes, your file name is confusing 
![image](https://user-images.githubusercontent.com/43061141/101029637-6531a780-3579-11eb-8aab-3ffee004f45a.png)

Reply:
No i did not.
I noticed the my programing language folder was missing from the egwoloOkorodudu folder on the submission folder
so i just added it back.
The task on this pull request is render the related product list on HTML page with JavaScript.
Thank you.

Reply:
Ok, But now the issue is that the file name is confusing. I am seeing `programmingLanguage1.txt and programmingLanguage1` instead of just `programmingLanguage1.txt`